app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -711,10 +711,12 @@ DeviceDelegateOpenXR::StartFrame(const FramePrediction aPrediction) {
     if (location.locationFlags & XR_SPACE_LOCATION_POSITION_VALID_BIT) {
       caps |= m.Is6DOF() ? device::Position : device::PositionEmulated;
     }
-    m.immersiveDisplay->SetCapabilityFlags(caps);
 
     // Update WebXR room scale transform if the device supports stage space
     if (m.stageSpace != XR_NULL_HANDLE) {
+      // Add 'stage' capability (this enables 'bounded' reference spaces)
```

Comment:
I don't want to be superpicky but please remove this comment before landing, as the code is really self-explanatory.

Reply:
Will do, thanks :)